clean.cmd

```diff
@@ -8,12 +8,13 @@ for /f "tokens=2 delims=," %%F in ('tasklist /nh /fi "imagename eq VBCSCompiler.
 :: compare to match as many versions of "all" that we can
 :: All other argument validation happens inside Run.exe
 set NO_DASHES_ARG=%1
+if not defined NO_DASHES_ARG goto no_args
 if /I [%NO_DASHES_ARG:-=%] == [all] (
   echo Cleaning entire working directory ...
   call git clean -xdf
   exit /b !ERRORLEVEL!
 )
-
+:no_args
 if [%1]==[] set __args=-b
```

Comment:
I can if you want, that involves a bit of refactoring the way the script's already written since it ends up calling  clean %__args% %*.  This means it expects __args to be only set if there are no parameters, so setting it to a value (to allow - substitution to work) would cause it to provide the arg twice.